registrations/management/commands/update_initial_sequence.py

```diff
@@ -0,0 +1,83 @@
+from os import environ
+
+from django.core.management.base import BaseCommand, CommandError
+
+from registrations.models import SubscriptionRequest
+
+from seed_services_client import StageBasedMessagingApiClient
+
+from ._utils import validate_and_return_url
+
+
+class Command(BaseCommand):
+    help = ("This command will loop all subscription requests and find the "
+            "corresponding subscription in SBM and update the "
+            "initial_sequence_number field, we need this to fast forward the "
+            "subscription.")
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--sbm-url', dest='sbm_url', type=validate_and_return_url,
+            default=environ.get('STAGE_BASED_MESSAGING_URL'),
+            help=('The Stage Based Messaging Service to verify '
+                  'subscriptions for.'))
+        parser.add_argument(
+            '--sbm-token', dest='sbm_token', type=str,
+            default=environ.get('STAGE_BASED_MESSAGING_TOKEN'),
+            help=('The Authorization token for the SBM Service'))
+
+    def handle(self, *args, **kwargs):
+        sbm_url = kwargs['sbm_url']
+        sbm_token = kwargs['sbm_token']
+
+        if not sbm_url:
+            raise CommandError(
+                'Please make sure either the STAGE_BASED_MESSAGING_URL '
+                'environment variable or --sbm-url is set.')
+
+        if not sbm_token:
+            raise CommandError(
+                'Please make sure either the STAGE_BASED_MESSAGING_TOKEN '
+                'environment variable or --sbm-token is set.')
+
+        sbm_client = StageBasedMessagingApiClient(sbm_token, sbm_url)
+
+        sub_requests = SubscriptionRequest.objects.all()
+
+        updated = 0
+        for sub_request in sub_requests:
+            subscriptions = sbm_client.get_subscriptions({
+                'identity': sub_request.identity,
+                'created_at__gt': sub_request.created_at
```

Comment:
I think we should also make sure the `message_set` is the same for the request and the subscription